### ISSUE TITLE
Convert documentation

### ISSUE THREAD

**Issue by bre7** [drkameleon/iswift#49]
Example:
 ```objectivec
/**
 *  yyyyyyyyyyy.
 *
 *  @param color zzzzzzzzzz
 *
 *  @return xxxxxxx
 */
```

Output:
```swift
/**
 *  yyyyyyyyyyy.
 *
 *  - param: color zzzzzzzzzz
 *
 *  - return: xxxxxxx
 */
```

**Comment by drkameleon** [comment]
Yep, I've thought of that too. Nice idea. I'll put it in my to-do list. ;)

**Comment by drkameleon** [comment]
Implemented as of the upcoming 1.6 (as initially suggested).

Also related: https://github.com/drkameleon/iswift/issues/56
